Content.Tests/Shared/Chemistry/SolutionTests.cs

```diff
@@ -1,476 +0,0 @@
-using Content.Shared.Chemistry.Components;
```

Comment:
Why were all of these tests removed?

Reply:
Because solutions now need to get prototype info on boiling points and such, and it kept throwing "prototypes not loaded" errors. Also, I have no idea on how to make these tests work. I've asked on #howdoicode but got no answers. If someone helps me out I'll gladly put them back.